js/scripts.js

```diff
@@ -0,0 +1,17 @@
+/*scripts.js
+Conditional instructions in practice*/
+
+var a = 3;
+var b = 6;
+var value = (a * a) - (2 * a * b) + (b * b);
+console.log("Value= " + value);
+
+if (value > 0) {
+    console.log("Wynik dodatni")
+}
+else if (value < 0) {
+    console.log("Wynik ujemny")
+}
+else {
```

Comment:
are you sure that in this case value will be always equal to `0`? what will happen when `a` or `b` will be a letter?